.pre-commit-config.yaml

```diff
@@ -110,7 +115,7 @@ repos:
         files: ^tests/
         args:
           [
-            '--ignore=D103,D213',
+            '--add-ignore=D103,D107,D213',
```

Comment:
are these needed here still if these are already defined in pyproject.toml

Reply:
will check